fst_lookup/fst.py

```diff
@@ -210,35 +213,46 @@ def __init__(
         self.arcs_from = arcs_from
 
     def __iter__(self) -> Iterator[RawTransduction]:
-        yield from self._accept(self.initial_state, [], [{}])
+        yield from self._accept(self.initial_state, [], [{}], [])
 
     def _accept(
         self,
         state: StateID,
         transduction: List[Symbol],
         flag_stack: List[Dict[str, str]],
+        stems: List[Symbol]
     ) -> Iterable[RawTransduction]:
```

Comment:
`RawTransduction` must be changed such that it includes `stems` if this PR is to be accepted.

Reply:
I changed the typing and tested them with `mypy`.